__tests__/components/Comments/WriteComment.test.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import {render, cleanup, fireEvent} from '@testing-library/react-native';
 import WriteComment, {styles} from '../../../src/components/Comments/WriteComment';
 import Store from '../../../src/context/Store';
-import {fonts} from '../../../src/utils/fonts';
+import {fonts, normalize} from '../../../src/utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `normalize` from the `fonts` module. Ensure that this function is used appropriately in the rest of the code.
```diff
- import {fonts} from '../../../src/utils/fonts';
+ import {fonts, normalize} from '../../../src/utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->